src/parser/monk/mistweaver/modules/spells/ExpelHarm.tsx

```diff
@@ -1,41 +1,40 @@
 import SPELLS from 'common/SPELLS';
-import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
-import Events from 'parser/core/Events';
-
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { HealEvent } from 'parser/core/Events';
 
 class ExpelHarm extends Analyzer {
 
-    selfHealing = 0;
-    selfOverheal = 0;
-    targetHealing = 0;
-    targetOverheal = 0;
-    gustsHealing = 0;
-    lastCastTarget = null;
-    sourceTarget = null;
-    numberToCount = 0;
-
-    constructor(...args) {
-        super(...args);
+    selfHealing: number = 0;
+    selfOverheal: number = 0;
+    targetHealing: number = 0;
+    targetOverheal: number = 0;
+    gustsHealing: number = 0;
+    lastCastTarget: number = 0;
+    sourceTarget: number = 0;
```

Comment:
For clarity, might these be better as -1 or something (sourceTarget / lastCastTarget)? I dont know if combatant ids are indexed starting at 0, but if they are it would feel weird to make the default 0, which might actually refer to an existing enemy or combatant